lemminx-liberty/src/test/java/io/openliberty/LibertyDiagnosticTest.java

```diff
@@ -434,7 +434,7 @@ public void testInvalidPlatformDiagnostic() throws BadLocationException {
                 "<server description=\"Sample Liberty server\">", //
                 "       <featureManager>", //
                 "               <feature>batch-1.0</feature>", //
-                "               <platform>jaX</platform>", //
+                "               <platform>javaee</platform>", //
```

Comment:
Change this to mixed case to ensure code handles it `javaEE`.

Reply:
This test is rolled back as this server.xml already contains most of the platform types.
Added a new test to validate
1. case insensitive matching
2. filter for removing existing platforms
3. filter for removing conflicting platforms